torchaudio/transforms/_transforms.py

```diff
@@ -599,11 +597,17 @@ def __init__(
         wkwargs: Optional[dict] = None,
         center: bool = True,
         pad_mode: str = "reflect",
-        onesided: bool = True,
```

Comment:
We need to keep the argument here, otherwise client code will break if it gives all the arguments as positional arguments.

Some strategy we employed in the past is to change the default value to None first, and issue a warning if the actual value is not None.
Then once we complete one release, we actually remove it.